### PR TITLE
add mochi

### PR DESCRIPTION
**Pull request by andrewc910**
## Link
https://github.com/gitlato/mochi

## Checklist
* [ ] - Shard is at least 30 days old.
- Currently 16 days
* [x] - Shard has CI implemented.
* [x] - Shard has daily/weekly periodic builds (ideally with Crystal latest and nightly).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [graphql-crystal](https://github.com/ziprandom/graphql-crystal) - [Graphql](http://graphql.org) implementation
  * [kave](https://github.com/jwoertink/kave) - Kemal API Version Extension
  * [kemal-session](https://github.com/kemalcr/kemal-session) - Session handler for Kemal
+ * [mochi](https://github.com/gitlato/mochi) - Authentication shard with authentication, confirmation, oauth, invites & more (inspired by devise)
```
Comment (veelenga) [inline README.md]: "Authentication shard with authentication" sounds strange. Would you mind to re-phase it.

Comment (andrewc910) [reply README.md]: Lol yeah that sounded pretty bad.

**Comment by andrewc910** [comment]
How this version?

File: README.md [hunk 2]
```diff
@@ -263,6 +263,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [graphql-crystal](https://github.com/ziprandom/graphql-crystal) - [Graphql](http://graphql.org) implementation
  * [kave](https://github.com/jwoertink/kave) - Kemal API Version Extension
  * [kemal-session](https://github.com/kemalcr/kemal-session) - Session handler for Kemal
+ * [mochi](https://github.com/gitlato/mochi) - Authentication shard inspired by Devise. Includes Authenticable, Confirmable, Invitable & more.
```
Comment (Blacksmoke16) [inline README.md]:
```suggestion
 * [mochi](https://github.com/gitlato/mochi) - Authentication shard inspired by Devise supporting: Authenticable, Confirmable, Invitable & more.
```

Makes it a single sentence like majority of other shards, and removed the ending period for consistency.

**Comment by Blacksmoke16** [comment]
@andrewc910 Can you ensure you have a periodic build setup?

**Pull request merged**